README.md

```diff
@@ -9,10 +9,7 @@ This is an [Arduino Library](https://www.arduino.cc/en/Guide/Libraries) for Ardu
 * Install the library by [Using the Library Manager](https://www.arduino.cc/en/Guide/Libraries#toc3)
 * **OR** by [Importing the .zip library](https://www.arduino.cc/en/Guide/Libraries#toc4) using either the [master](https://github.com/TheThingsNetwork/arduino-device-lib/archive/master.zip) or one of the [releases](https://github.com/TheThingsNetwork/arduino-device-lib/releases) ZIP files.
 
-To use build in messages, thanks to [Protobufs](https://en.wikipedia.org/wiki/Protocol_Buffers), in the Arduino library, you might want to install the Nanopb compiler by [downloading](http://koti.kapsi.fi/%7Ejpa/nanopb/download/) it.
-After that, in your shell, just define `$NANOPB` pointing to the Nanopb compiler's path e.g.:
-
-`export NANOPB="~/dev/bin/nanopb-0.3.6-macosx-x86"`
+To use built-in messages in the Arduino library, you might want to [download](http://koti.kapsi.fi/%7Ejpa/nanopb/download/) the [Nanopb compiler](https://koti.kapsi.fi/jpa/nanopb/).
```

Comment:
Again, compilation is only necessary for regenerating the protobufs. This library should be stand-alone.

Apart from this basic functionality (standardizing sensor data), we might need a guide on how to generate code from custom protos, but that is beyond the scope.